Replace "-" on precision support page (#4429)
ROCm/ROCm: docs/reference/precision-support.rst

  * In some AMD documents and articles, float8 (E5M2) is referred to as bfloat8.

  * The :doc:`low precision floating point types page <hip:reference/low_fp_types>`
    describes how to use these types in HIP with examples.

> Level of support definitions

Reviewer: ```suggestion
Support level definitions
```

  * Any type can be emulated by software, but this page does not cover such
    cases.

> Data type support by Hardware Architecture

Reviewer: ```suggestion
Data type support by hardware architecture
```

Reviewer: I'm not sure why you rephrased this.

==========================================

The MI200 series GPUs, which include MI210, MI250, and MI250X, are based on the

> CDNA2 architecture. The MI300 series GPUs, consisting of MI300A, MI300X, and
> MI325X, are based on the CDNA3 architecture.

Reviewer: ```suggestion
CDNA2 architecture. The MI300 series GPUs, which includes of MI300A, MI300X, and
MI325X, are based on the CDNA3 architecture.
```

        - ✅/✅
        - ✅/✅

.. note::
   
  As random number generation libraries, rocRAND and hipRAND only specify output
  data types for the random values they generate, with no need for input data
  types.

> Libraries internal calculations type support

Author: There is no hyperlink on this line. Could you help me out where should be hyperlink?
